services/core-api/app/api/mines/incidents/resources/mine_incidents.py

```diff
@@ -325,6 +325,10 @@ def put(self, mine_guid, mine_incident_guid):
                     incident.send_awaiting_final_report_email(True)
                     incident.send_awaiting_final_report_email(False)
                     trigger_notifcation(f'A new Mine Incident has been created for ({incident.mine_name})', incident.mine_table, 'MineIncident', incident.mine_incident_guid, {})
+                if value == 'FRS':
```

Comment:
Will these status based emails get sent every time it is one of those stauses(ie. updating another field with the same status hits this endpoint). Should it only be when the existing status code does not match the request one?

Reply:
at the moment it is whenever that status is set. I think when we actually wire in all the logic around the statuses this might need to be changed cause we need that logic in first.